### DISCUSSION TITLE
2006/08/07/labview-%E4%B8%AD%E7%9A%84%E6%95%B0%E5%AD%97%E5%9E%8B%E6%95%B0%E6%8D%AE-1-%E6%8E%A7%E4%BB%B6%E5%92%8C%E5%B8%B8%E9%87%8F

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#103] [Announcements]
# 2006/08/07/labview-%E4%B8%AD%E7%9A%84%E6%95%B0%E5%AD%97%E5%9E%8B%E6%95%B0%E6%8D%AE-1-%E6%8E%A7%E4%BB%B6%E5%92%8C%E5%B8%B8%E9%87%8F

https://lv.qizhen.xyz/#docs/data\number

https://blog.qizhen.xyz/2006/08/07/labview-%E4%B8%AD%E7%9A%84%E6%95%B0%E5%AD%97%E5%9E%8B%E6%95%B0%E6%8D%AE-1-%E6%8E%A7%E4%BB%B6%E5%92%8C%E5%B8%B8%E9%87%8F

**Comment by ruanqizhen** [comment]
之前博客下的留言：

刘伟印
2008年08月21日 15:41 编辑
溢出错误确实很难找，呵呵

回复
fx
2006年08月12日 00:38 编辑
太专业了,看不懂.
 

回复
清扬
2006年08月8日 09:28 编辑
很细节的东西,看来我也要注意注意了.labview有自定义控件的功能,如果要做漂亮的界面的话,可以考虑用上那个功能.
